File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] arr) {
       int n = arr.length;
        
        for (int i = 0; i < n-1; i++) {
            for(int j=i+1; j>0; j--){
                if (arr[j]<arr[j-1]) {
                    int t = arr[j];
                    arr[j] = arr[j-1];
                    arr[j-1] = t;
                }
            }
        }
    } 
    
}

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        r  = 0
        l = len(nums) - 1
        mid = 0
        while(mid <= l):
            if nums[mid] == 0:
                temp = nums[mid]
                nums[mid] = nums[r]
                nums[r] = temp
                r+=1
                mid +=1
            elif nums[mid] == 1:
                mid += 1
            elif nums[mid] == 2:
                temp = nums[mid]
                nums[mid] = nums[l]
                nums[l] = temp
                l -= 1
        